Tests/LibWeb/Text/expected/HTML/Location-has-unforgeable-properties.txt

```diff
@@ -0,0 +1,156 @@
+href exists on location itself: true
+href does not exist on LocationPrototype: true
+href descriptor is not undefined: true
+href enumerable: true
+href configurable: true
```

Comment:
I think `configurable` should be false for all of these properties. This test also shows the same issue I believe:  http://wpt.live/html/browsers/the-window-object/proxy-getOwnPropertyDescriptor.html   

I was able to get this output to match Chrome and Firefox and the above WPT to pass by moving `LocationPrototype::define_unforgeable_attributes()` to the bottom of `Location::initialize()`, which I assume is correct?

Reply:
I believe it's related to this FIXME:
https://github.com/LadybirdBrowser/ladybird/blob/b8adf193f52c4c7089e4e97edd380db189bd6362/Libraries/LibWeb/HTML/Location.cpp#L527-L533

Reply:
I'm happy to merge this as-is, since this difference isn't related to the implementation of `[LegacyUnforgable]`. Making `Location` behave as it does in other browsers can be done in a future PR.